Chapter2_ProgramStructure/Chapter 2 #1.JS

```diff
@@ -0,0 +1,14 @@
+function f1()
```

Comment:
Worth going through the AirBnb style guide rules:
https://github.com/airbnb/javascript